e2e/smokeTest.spec.js

```diff
@@ -0,0 +1,30 @@
+import { test, expect } from '@playwright/test'
```

Comment:
It would be nice to have in readme how this test can be run locally and how to install playwright dependencies.

Reply:
Added into Readme.